core/ui/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/ui/date/relativeTime.kt

```diff
@@ -27,7 +27,8 @@ fun getRelativeTime(
     to: Instant,
     clock: Clock = Clock.System,
     formatSeconds: Boolean = false,
-    showDate: Boolean = true
+    showDate: Boolean = true,
+    showDateAndTime: Boolean = false
 ): CharSequence {
```

Comment:
Im really struggling with this method and the boolean toggles tbh, I think its very hard to understand what the method emits in what situation. But I could also not find a truly satisfying solution for it. Like, what should I expect if I pass showDateAndTime = true, but showDate = false to the function? Maybe a enum? I really dont know. 
Overall the function just feels hard to read. Maybe a little bit of refactoring already helps, or maybe we just have to live with it, idk.
What are your thoughts on this?